workspaces/copilot/plugins/copilot/src/components/Cards/DashboardCards.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { PropsWithChildren } from 'react';
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import AssessmentIcon from '@mui/icons-material/Assessment';
+import ThumbUpIcon from '@mui/icons-material/ThumbUp';
+import CodeIcon from '@mui/icons-material/Code';
+import Box from '@mui/material/Box';
+import { Card } from './Card';
+import { CardsProps } from '../../types';
+import { styled } from '@mui/material/styles';
+import { Metric } from '@backstage-community/plugin-copilot-common';
+
+const CardBox = styled(Box)(() => ({
```

Comment:
There seems to be something about this way of styling components that seems to "leak" out from this plugin and affect unrelated components in the rest of the instance that it is installed into.

Im not an expert in this so I can't say what it is. @awanlin can you shed some light on this?